building/build_from_commandline.adoc

```diff
@@ -0,0 +1,30 @@
+#  Building locally via commandline
+
+First, fetch the sources of the project you want to build from Github. For details, see link:configuring_git_workflow.adoc#Sources[Configuring Git Workflow - Sources].
+
+Then, switch into the directory where you checked out the sources, and run this:
+
+
+[source,bash]
+----
+cd ~/my-github-stuff/jbosstools-base
+
+# build everything and run all tests
```

Comment:
-e is also not really something to suggest to add manually every time, this is more if trying to debug maven plugins.